.github/workflows/rules.yml

```diff
@@ -1,26 +1,30 @@
-name: Rules 
+name: Rules
 
 on:
 
   workflow_call:
     inputs:
         os:
-          required: true 
-          type: string 
+          required: true
+          type: string
         python-version:
-          required: true 
+          required: true
           type: string
         environment:
-          required: true 
-          type: string 
+          required: true
+          type: string
         pip-extras:
-          required: false 
-          type: string 
-          default: " "
+          required: false
+          type: string
+          default: ""
         doctests:
-          required: false 
+          required: false
+          type: boolean
+          default: false
+        disable-jit:
```

Comment:
As written below, this should not be required

Reply:
More explicitly, we should perform tests with and without JIT enable, to make sure the code is compiling fine (JIT enabled) but at the same time upload the coverage report (with JIT disabled).

Reply:
I have included `pytest-env`, at the qibo is already testing compiled code.